adafruit_wiznet5k/adafruit_wiznet5k_socket.py

```diff
@@ -88,17 +139,28 @@ def is_ipv4(host):
 
 # pylint: disable=invalid-name, too-many-public-methods
 class socket:
-    """A simplified implementation of the Python 'socket' class
-    for connecting to a Wiznet5k module.
-
-    :param int family: Socket address (and protocol) family.
-    :param int type: Socket type.
+    """
+    A simplified implementation of the Python 'socket' class for connecting
+    to a Wiznet5k module.
     """
 
     # pylint: disable=redefined-builtin,unused-argument
     def __init__(
-        self, family=AF_INET, type=SOCK_STREAM, proto=0, fileno=None, socknum=None
-    ):
+        self,
+        family: int = AF_INET,
+        type: int = SOCK_STREAM,
+        proto: Any = 0,
+        fileno: Optional[int] = None,
+        socknum: Optional[int] = None,
```

Comment:
`fileno` and `socknum` are typed `int` here in the arguments. But down a few lines in the docstring they are `Any`. 

Since they are unused it probably doesn't matter too much, but I think it'd be best to make them consistent. Can you change one or the other so that they match?